cli/cppcheckexecutorsig.cpp

```diff
@@ -49,7 +49,7 @@
 #   include <ucontext.h>
 #endif
 
-// TODO: __USE_DYNAMIC_STACK_SIZE is depedent on the features.h include and not a built-in compiler define, so it might be problematic to depedent on it
+// TODO: __USE_DYNAMIC_STACK_SIZE is dependent on the features.h include and not a built-in compiler define, so it might be problematic to depedent on it
```

Comment:
nit: another depedent

Reply:
Done. Also found the same typo in a third place.